java/viewshed-camera/src/main/res/values/strings.xml

```diff
@@ -1,8 +1,4 @@
 <resources>
     <string name="app_name">Viewshed Camera</string>
     <string name="update_from_camera">Update from camera</string>
-    <string name="brest_dtm">https://scene.arcgis.com/arcgis/rest/services/BREST_DTM_1M/ImageServer</string>
-    <string name="brest_building_layer">
-        https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/layers/0
```

Comment:
my only thought here is wondering why the new Girona string isn't going into the resources folder as the Brest one was previously (it's now directly in the main java class)? I don't think it really matters, but just curious if it was Android/Android samples specific pattern to leave them in here then we could do that.

Reply:
So Android Studio linter used to complain about all strings in code not being in the `strings.xml` file. This was to do with localization (where translators would theoretically provide alternatives for different spoken languages). That's not a requirement for URLs, so where possible I'm trying to migrate URLs from `strings.xml` into the code directly (as it's also more readable that way)